crates/cli/tests/cli_run.rs

```diff
@@ -106,6 +107,11 @@ mod cli_run {
         assert_multiline_str_eq!(err.as_str(), expected);
     }
 
+    fn assert_valid_roc_check_status(status: ExitStatus) {
```

Comment:
This was necessary because `roc check` was returning exit code 2 when it previously returned 0 for tests using `basic-cli` v0.12.0 because of backpassing deprecation warnings. This unblocks us on making this change, instead of making a new `basic-cli` release to test.